vmdb/db/migrate/20150111161446_create_nodes.rb

```diff
@@ -0,0 +1,17 @@
+class CreateNodes < ActiveRecord::Migration
+  def up
+    create_table :container_nodes do |t|
+      t.string    :ems_ref
+      t.string    :name
+      t.datetime  :creation_timestamp
+      t.string    :resource_version
+      t.bigint    :ems_id
+    end
+    add_index "container_nodes", ["ems_id"], :name => "index_container_nodes_on_ems_id"
```

Comment:
- Use symbols for consistency
- The Array is not needed on the seconds parameter
- The `:name` option is not needed at all